R/class-plotdata.R

```diff
@@ -45,6 +47,17 @@ newPlotdata <- function(.dt = data.table(),
   lat <- veupathUtils::findColNamesFromPlotRef(variables, 'latitude')
   lon <- veupathUtils::findColNamesFromPlotRef(variables, 'longitude')
 
+  # If we ask for the point ids, ensure the column is present. Otherwise set to null. 
+  if (!is.null(returnPointIds) && returnPointIds && length(idColumn) > 0) {
+    if (idColumn %in% names(.dt) && nrow(.dt) == uniqueN(.dt[[idColumn]])) {
+      idCol <- idColumn
+    } else {
+      idCol <- NULL
```

Comment:
Hi! Should this maybe throw an error? Seems like there will be downstream trouble if we asked for point IDs and there's no column to get them from?

Reply:
That's a good idea! I'll add it.